.github/workflows/industrial_ci_action.yml

```diff
@@ -0,0 +1,16 @@
+name: CI
+
+on: [push, pull_request]
+
+jobs:
+  industrial_ci:
+    strategy:
+      matrix:
+        env:
+          - {ROS_DISTRO: melodic, ROS_REPO: testing}
+          - {ROS_DISTRO: melodic, ROS_REPO: main}
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - uses: 'ros-industrial/industrial_ci@master'
+        env: ${{matrix.env}}
```

Comment:
I believe that syntax isn't quite idiomatic for `matrix` in GHA: https://github.com/ros-industrial/industrial_ci/issues/590.

That issue also documents the alternative structure which is.

Reply:
Thanks, that is actually nicer!